app/gui/src/main/scala/org/bitcoins/gui/dialog/SendDialog.scala

```diff
@@ -0,0 +1,58 @@
+package org.bitcoins.gui.dialog
+
+import scalafx.Includes._
+import scalafx.application.Platform
+import scalafx.geometry.Insets
+import scalafx.scene.control.{ButtonType, Dialog, Label, TextField}
+import scalafx.scene.layout.GridPane
+import scalafx.stage.Window
+
+object SendDialog {
+
+  def showAndWait(parentWindow: Window): Option[(String, String)] = {
+    val dialog = new Dialog[Option[(String, String)]]() {
+      initOwner(parentWindow)
+      title = "Send"
+    }
+
+    dialog.dialogPane().buttonTypes = Seq(ButtonType.OK, ButtonType.Cancel)
+
+    val addressTF = new TextField()
+    val amountTF = new TextField()
+
+    dialog.dialogPane().content = new GridPane {
+      hgap = 10
+      vgap = 10
+      padding = Insets(20, 100, 10, 10)
+
+      var nextRow: Int = 0
+      def addRow(label: String, textField: TextField): Unit = {
+        add(new Label(label), 0, nextRow)
+        add(textField, 1, nextRow)
+        nextRow += 1
+      }
+
+      addRow("Receiving Address", addressTF)
```

Comment:
I think "Receiving Address" is a little confusing, it can sound like we are receiving the funds. I think it would be better to just do "Address"